Sming/Wiring/Print.h

```diff
@@ -26,12 +26,14 @@
 
 #define INITIAL_PRINTF_BUFFSIZE 128
 
+#undef printf_P
```

Comment:
Yes, I guess if printf / m_printf will be progmem safe, then no need for _P variant. For printf I do not care about speed so much, but sprintf I would keep it fast